collector/collector.go

```diff
@@ -34,6 +34,14 @@ func getValidLabelName(str string) string {
 	return prometheusMetricNameInvalidCharactersRegex.ReplaceAllString(strings.Replace(str, "-", "_", -1), "")
 }
 
+// A boolean cannot be directly converted to a number
+func boolToFloat64(boolean bool) (value float64) {
```

Comment:
This is a tad verbose but also self explanatory trick. A single conditional `return` would have felt lame :)